src/config/config.yml

```diff
@@ -15,6 +27,8 @@ invoke:
       getDefects:
         name: cvs-svc-defects
         mock: tests/resources/defects.json
+      getIvaDefectsByManual:
+        name: cvs-svc-defects
   remote:
```

Comment:
I know there are plans to eventually move towards SAM but whilst we are still using serverless, particularly for local invoke I think it's worth including getIvaDefects in here.

Reply:
Are we ever going to want to invoke it and not just call the API?

Reply:
There may come a time when a BE service like cert-gen may want to invoke it potentially, I know for the majority of flows BE services to lambda-to-lambda invocations over API calls.